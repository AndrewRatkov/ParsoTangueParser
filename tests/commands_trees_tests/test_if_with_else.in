if 30 == 2 * 3 * 5 * 1 then str myNick := "hrundelair"; else str myNick := "clown"; fi