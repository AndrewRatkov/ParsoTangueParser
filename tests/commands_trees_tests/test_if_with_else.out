IF
|
+--->[INT,==]------->[INT,30]
|        |
|        +---------->[INT,* ]------->[INT,* ]------->[INT,* ]------->[INT,2]
|                        |               |               |
|                        |               |               +---------->[INT,3]
|                        |               |
|                        |               +---------->[INT,5]
|                        |
|                        +---------->[INT,1]
|
THEN
|
+--->[DEF str myNick]--->[STR,"hrundelair"]
|
ELSE
|
+--->[DEF str myNick]--->[STR,"clown"]
|
FI
